Makefile.dep

```diff
@@ -103,6 +103,7 @@ endif
 
 ifneq (,$(filter gnrc_sixlowpan_nd_router,$(USEMODULE)))
   USEMODULE += gnrc_sixlowpan_nd
+  USEMODULE += gnrc_ndp_router
```

Comment:
That was my thought, too, but a) where else are router advertisements sent and b) why else is RFC6775 mentioned there?